core/src/mindustry/logic/LExecutor.java

```diff
@@ -260,6 +261,19 @@ public void run(LExecutor exec){
                             res = b == null ? null : b.tile;
                             build = true;
                         }
+                        case block -> {
+                            if(exec.obj(block) instanceof Block) {
+                                if (exec.bool(enemy)) {
+                                    Building b = Units.findEnemyTile(unit.team, unit.x, unit.y, Float.MAX_VALUE, e -> !e.dead() && e.block == exec.obj(block));
+                                    res = b == null ? null : b.tile;
+                                    build = true;
+                                } else {
```

Comment:
[Read the style guide!](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md#style-guidelines) I'm getting sick of saying this.

Reply:
im very sorry, fixed